src/utils/order.js

```diff
@@ -0,0 +1,6 @@
+export const calculateSum = (order) => order.length
+? order
+    .map(({ price, quantity }) => price * quantity)
+    .reduce((x, y) => x + y)
+    .toFixed(2)
```

Comment:
Consider using lodash and this conditional approach to refactor this a bit:
```suggestion
if (!order.length) {
 return 0
}

return sumBy(order, ({ price, quantity }) => price * quantity)
```